enterprise/signals/gp_signals.py

```diff
@@ -513,14 +513,15 @@ def WidebandTimingModel(
     dmefac=parameter.Uniform(pmin=0.1, pmax=10.0),
     log10_dmequad=parameter.Uniform(pmin=-7.0, pmax=0.0),
     dmjump=parameter.Uniform(pmin=-0.01, pmax=0.01),
-    dmefac_selection=Selection(selections.no_selection),
-    log10_dmequad_selection=Selection(selections.no_selection),
+    selection=Selection(selections.no_selection),
     dmjump_selection=Selection(selections.no_selection),
     dmjump_ref=None,
     name="wideband_timing_model",
 ):
     """Class factory for marginalized linear timing model signals
-    that take wideband TOAs and DMs.  Currently assumes DMX for DM model."""
+    that take wideband TOAs and DMs. DM noise can be adjusted in analogy to
+    the tempo/tempo2/pint parameter convention, variance = dmefac^2 (dmerr^2 + equad^2).
```

Comment:
should this be dmequad^2?

Reply:
Yes! While fixing that I also expanded the docstring.